assembl/static2/js/app/pages/debate.jsx

```diff
@@ -30,11 +34,41 @@ class Debate extends React.Component {
   displayThumbnails() {
     this.setState({ isThumbnailsHidden: !this.state.isThumbnailsHidden });
   }
+  redirectToV1Threads() {
```

Comment:
@paolina92 I see your point. Very well, if you're not going to put it in`utils` since it's not UI interactions, then sure, `main.jsx` it is. You can pass it as a prop to all children, that is the react way, no?